style/care.css

```
@@ -0,0 +1,69 @@
+
+.intro {
+    margin-top: 1em;
+    height: 275px;
+    display: flex;
+    width:100%;  
+}
+
+#summary {
+    width:55%;
+    background-color: #436105;
+    background-image: url(../assets/intro.jpg);
+    background-size: cover;
+
+
+}
+
+#summary p{
+    display: block;
+    height: 10px;
+    width: 400px;
+    margin-left: auto;
+    margin-right: 1em;
+    background-color: white;
+    padding: 1em;
+}
+
+#table-of-contents{
+    background-color: white;
+    padding-left: 20px;
+    width: 50%;
+}
+
+#table-of-contents h2 {
+    margin: 0;
+}
+
+.content {
+    margin-left: 1em;
+    /* background-color: red; */
+    max-width: 1000px;
+    margin: .5em auto;
+    padding: .5em 2em;
+    display: flex;
+    flex-direction: column;
+}
+
+#Light, #Watering, #Soil, #Troubleshooting {
+    display: flex;
+    flex-wrap: wrap;
+    /* align-items: flex-start; */
+    justify-content: space-around;
+    gap: 20px;
+    /* background-color: red; */
+    margin: 1em;
```

```suggestion
    flex-wrap: wrap;
    justify-content: space-around;
    gap: 20px;
    margin: 1em;
```

Like in our Python projects, commented code should be removed. Leaving commented out question might beg the question "why is this here? was it needed? should it be added back in?" For clarity and keeping your codebase clutter free, remove commented out code.